### PR TITLE
Review parser - attempt to structure address data.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
This PR attempts to provide a model for an Address broken down into specific parts. It does so by taking the parsed address we believe we have for a restaurant and using it to query Google's geocoding api. It then looks at the data retrieved from Google and decides whether the address parts are likely to be accurate or not for the given address.

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

File: src/main/scala/com/gu/restaurant_review_parser/parsers/Delimiters.scala [hunk 1]
```diff
@@ -3,8 +3,9 @@ package com.gu.restaurant_review_parser.parsers
 object Delimiters {
 
   val CommaDelimiter = ","
-  val PipeDelimiterWithSpaces = " \\| "
+  val PipeDelimiterWithSpaces = "\\s\\|\\s"
```
Comment (mchv) [inline src/main/scala/com/gu/restaurant_review_parser/parsers/Delimiters.scala]: you should use `"""` (triple quotes) so you don't have to escape manually

Comment (mchv) [inline src/main/scala/com/gu/restaurant_review_parser/parsers/Delimiters.scala]: `PipeDelimiterWithSpaces` will become:
```scala
val PipeDelimiterWithSpaces = """\s\|\s"""
````

**Comment by mchv** [comment]
Nice code 👍

**Pull request merged**